### ISSUE TITLE
Language with locale in URL

### ISSUE THREAD

**Issue by hamzstramgram** [rappasoft/laravel-5-boilerplate#368]
What do you guys think about having this package integrated also?
[https://github.com/mcamara/laravel-localization](https://github.com/mcamara/laravel-localization)

For the frontend part it would be great to have the locale appearing in the URL for SEO purpose.

**Comment by blomdahldaniel** [comment]
It seems like a very good project. But in some ways it might not be necessary for a lot of people as something default in this project. As of now I think a lot of people take one of the built in languages and set it to be default. 

So, adding this package would force more users to stuff that they might not need. And this boilerplate is mainly focused on the administration-part, not the front-end which the localization-package is built for. :)
So I think that it is a good package but I think that it should be added in by the devs themselves. This repo is, and should stay as, some sort of a lightweight boilerplate. Do you understand what I mean?

**Comment by rappasoft** [comment]
I like the idea of SEO friendly lang url's but I don't think that is the best package to do it. I tried to do it myself but it then adds an extra segment and renders all routes dead. So I don't know of a different way to do it.

**Comment by ninjaparade** [comment]
I've seen this package and it confuses me. For starters, say you have an english page at this url 
`http://website.com/about-us`

if you visit the french page, do you really think going to 
`http://website.com/fr/about-us` makes any sense? Or would it make more sense to end up at 

`http://website.com/fr/a-props`
 or something similar.

How do you guys normally handle this?

**Comment by rappasoft** [comment]
That makes sense to me, but that requires a language file for all route segments now.

**Comment by ninjaparade** [comment]
ya for sure, that's why I think the package linked doesn't really adhere to what I would consider `best practices` 

I've built a `pages` module that works in your boilerplate that handles named routes and routes in general in the manner I described above. Its a lot more complicated and personal problem to solve and I think it would be really challenging for you to add that into the boilerplate. Just my take though.
